src/mavlink-router/mainloop.cpp

```diff
@@ -742,6 +742,18 @@ bool Mainloop::_retry_timeout_cb(void *data)
 
 void Mainloop::_handle_pipe()
 {
+    enum CMD_ARG_INDEX {
+      CMD = 0,
```

Comment:
enum is a bit better, but actually best would be to use proper data structure with field names instead of std::vector<...> to pass the parsed config bits